x11-misc/slim/slim-1.3.9-r1.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake pam systemd
+
+DESCRIPTION="Simple Login Manager resurrected"
+HOMEPAGE="https://slim-fork.sourceforge.io/"
+SRC_URI="mirror://sourceforge/project/${PN}-fork/${P}.tar.gz"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="branding pam"
+
+RDEPEND="x11-libs/libXmu
+	x11-libs/libX11
```

Comment:
+ `x11-libs/libXext`

Reply:
I think that's only required for the "slimlock" tool, which is only built if PAM is enabled, so should this be `pam? (x11-lib/libXext)` ?

Reply:
Sure.  If it's used only with pam, add it conditionally.